.pre-commit-config.yaml

```diff
@@ -71,8 +71,39 @@ repos:
   rev: pylint-2.5.0
   hooks:
     - id: pylint
+      language_version: python3
+      args:
+      - --disable=import-error # E0401. Locally you could not have all imports.
+      - --disable=fixme # W0511. 'TODO' notations.
+
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: v0.770
+  hooks:
+    - id: mypy
       language_version: python3
       args: [
-        '--disable=import-error', # E0401. Locally you could not have all imports.
-        '--disable=fixme', # W0511. 'TODO' notations.
+        --ignore-missing-imports,
+        --disallow-untyped-calls,
+        --warn-redundant-casts,
       ]
+
+# pip3 install wemake-python-styleguide
+- repo: local
+  hooks:
+    - id: flake8
+      name: flake8
+      description: Python style guide enforcement
+      entry: flake8
+      args:
+        - --max-returns=2
+        - --max-arguments=4
+        # https://www.flake8rules.com/
+        # https://wemake-python-stylegui.de/en/latest/pages/usage/violations/index.html
+        - --ignore=
+                  WPS305, <!-- Found `f` string -->
+                  E501,   <!-- line too long (> 79 characters). Use 100 -->
+                  I001,   <!-- isort found an import in the wrong position. Use reorder-python-imports -->
+                  W503,   <!-- Deprecated rule. Use W504 instead. -->
```

Comment:
Use `extend-ignore` instead of `ignore` not to override the default ignores (where both W503 and W504 are ignored).